articles/lab-services/devtest-lab-guidance-prescriptive-adoption.md

```diff
@@ -33,5 +33,8 @@ There is a common set of concerns when migrating these workloads to the cloud:
 ## Intended audience
 The Enterprise-focused documentation is intended for IT planners, architects, and managers who are responsible for establishing and reviewing overall deployments and overseeing operations practices. As a result, this document emphasizes the overall process and recommended design principles to promote a secure and stable development/testing environment, which ultimately drives adoption of Azure DevTest Labs within an organization.
 
+## Enterprise Customers
```

Comment:
@tanmayeekamath -- Should "Customers" be lowercase to align with the other headers?

Reply:
Updated - thanks!